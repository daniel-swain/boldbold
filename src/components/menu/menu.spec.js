/* eslint-disable */
import React from 'react';
import Menu from './index';

describe('<Menu />', () => {
  let wrapper;

  beforeAll(() => {
    wrapper = shallow(<Menu />);
  });

  it('should render properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
