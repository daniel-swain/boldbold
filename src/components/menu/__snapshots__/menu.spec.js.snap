// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Menu /> should render properly 1`] = `
<div
  className="menu "
>
  <Header />
  <ul
    className="menu-actions"
  >
    <MenuItem
      func={[Function]}
      key="feed"
      name="feed"
    />
    <MenuItem
      func={[Function]}
      key="menu"
      name="menu"
    />
    <MenuItem
      func={[Function]}
      key="projects"
      name="projects"
    />
    <MenuItem
      func={[Function]}
      key="contact"
      name="contact"
    />
  </ul>
</div>
`;
